datafusion/src/logical_plan/expr.rs

```diff
@@ -1349,6 +1349,15 @@ pub fn unnormalize_cols(exprs: impl IntoIterator<Item = Expr>) -> Vec<Expr> {
     exprs.into_iter().map(unnormalize_col).collect()
 }
 
+/// Recursively un-alias an expressions
```

Comment:
The "recursively" part may be misleading, this function unwraps all current aliases

So an expr like `(a as "foo") + (b as "bar")` will not be unaliased, but an expr like `(a as "foo") as "bar"` will be unaliased to "foo"